templates/kubernetes/docs/1.23/charm-docker-registry.md

```diff
@@ -485,14 +485,13 @@ juju config docker-registry \
 
 The recommended way to delete images from the registry is to use the `rmi`
 action. If necessary, this charm can be configured to
-[allow deletion][storage-delete] of blobs and manifests by digest by setting
+allow deletion of blobs and manifests by digest by setting
```

Comment:
I think all of these pages have the same broken link for this section. Would you mind implementing the same fix for those as well?

Reply:
They didn't appear on the linkchecker but I will test this and make another PR for anything I find (need this one to land for some other stuff currently being done).